ui/app/AppLayouts/Wallet/stores/TokensStore.qml

```diff
@@ -56,8 +66,8 @@ QtObject {
                 expression: tokenIcon(model.symbol)
             },
             ExpressionRole {
-                name: "jsArraySources"
-                expression: model.sources.split(";")
+                name: "networkChainName"
```

Comment:
Is it pretended to be used instead of `chainName` role? If yes, then we should review other UI components.

Reply:
Ive removed it now